src/shared/api/api.ts

```diff
@@ -74,4 +75,9 @@ export const getApis = ({ callWithToken, callWithoutToken }: GetApisProps) =>
         path: 'pretotype/list',
         token,
       }),
+    'GET /company-list': () =>
+      callWithoutToken<SuccessResponse<CompanyListResponse[]>>({
```

Comment:
그리고 CompanyListResponse 뒤에 []가 붙는 게 좀 어색하네요. 이게 더 맞을 거 같습니다.
```suggestion
      callWithoutToken<SuccessResponse<CompanyResponse[]>>({
```